<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账号中心</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>561</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>账号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓氏</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>名字</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>账号id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>加群状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>群发状态</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>登录</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPhone">
      <property name="text">
       <string>Phone (带区号)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPhone"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelCode">
      <property name="text">
       <string>验证码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditCode"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelApiId">
      <property name="text">
       <string>API ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditApiId"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelApiHash">
      <property name="text">
       <string>API Hash:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditApiHash"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelTwoFaPassword">
      <property name="text">
       <string>二级密码:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditTwoFaPassword">
      <property name="placeholderText">
       <string>输入二级密码（如果有）</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonSendCode">
      <property name="text">
       <string>发送验证码</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButtonCodeLogin">
      <property name="text">
       <string>验证码登录</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButtonSessionLogin">
      <property name="text">
       <string>从Session登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>修改信息</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxModifyNickname">
      <property name="text">
       <string>修改名字</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>名字</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditFirstName"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxModifyLastName">
      <property name="text">
       <string>修改姓氏</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>姓氏</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditLastName"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxModifyAbout">
      <property name="text">
       <string>修改简介</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>简介</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditAbout"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxModifyAvatar">
      <property name="text">
       <string>修改头像</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonUpdateProfile">
      <property name="text">
       <string>更新资料</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
